quarkus-integration/quarkus/deployment/src/test/java/ai/timefold/solver/quarkus/testdata/multiple/constraintprovider/constraints/TestdataMultipleScoreConstraintProvider.java

```diff
@@ -0,0 +1,22 @@
+package ai.timefold.solver.quarkus.testdata.multiple.constraintprovider.constraints;
+
+import ai.timefold.solver.core.api.score.buildin.simple.SimpleScore;
+import ai.timefold.solver.core.api.score.stream.Constraint;
+import ai.timefold.solver.core.api.score.stream.ConstraintFactory;
+import ai.timefold.solver.core.api.score.stream.Joiners;
+import ai.timefold.solver.quarkus.testdata.multiple.constraintprovider.domain.TestdataMultipleScoreEntity;
+
+import org.jspecify.annotations.NonNull;
+
+public class TestdataMultipleScoreConstraintProvider extends TestdataAbstractMultipleScoreConstraintProvider {
```

Comment:
Still "multiple score".

Reply:
No reason for that, and it will be changed.